test.js

```diff
@@ -0,0 +1,10 @@
+/**
```

Comment:
no need to commit this file, try to review in VS code files before committing them